src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh

```diff
@@ -11,11 +11,16 @@ if [ "${CROMWELL_BUILD_PROVIDER}" = "${CROMWELL_BUILD_PROVIDER_TRAVIS}" ] && [ -
 
   prior_version=$(cromwell::private::calculate_prior_version_tag)
   export TEST_CROMWELL_PRIOR_VERSION_TAG="${prior_version}"
-  export TEST_CROMWELL_PRIOR_VERSION_CONF="papi_v2_${prior_version}_application.conf"
+  WOULD_BE_PRIOR_VERSION_CONF="papi_v2_${prior_version}_application.conf"
+  if [[ -f "$CROMWELL_BUILD_RESOURCES_DIRECTORY/$WOULD_BE_PRIOR_VERSION_CONF" ]]; then
```

Comment:
I'm pretty sure there's a shorthand for this kind of "get or default" elsewhere in this script

Reply:
There is but this is "file exists or default".